### PR TITLE
'async/await'

### PR DESCRIPTION
**Pull request by hhhluke**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/redis.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ function createClient(config, app) {
     });
   }
 
-  app.beforeStart(function* () {
-    const result = yield client.info();
+  app.beforeStart(async function () {
```
Comment (dead-horse) [inline lib/redis.js]: async () =>

**Comment by dead-horse** [comment]
https://github.com/eggjs/egg-redis/blob/master/package.json#L42 change to '8, 9' and reinstall node_modules to generate the new CI file

**Comment by hhhluke** [comment]
@dead-horse 集成没弄过不好意思，请问npm run ci 就行了吗？LF，CRLF和箭头函数通不过测试

**Comment by atian25** [comment]
这个是升级到 egg2 是吧，我来接手吧

**Comment by dead-horse** [comment]
我先合掉

**Pull request merged**